pkg/kubectl/cmd/clusterinfo.go

```diff
@@ -43,6 +43,7 @@ func NewCmdClusterInfo(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 		},
 	}
 	cmdutil.AddInclude3rdPartyFlags(cmd)
+	cmd.AddCommand(NewCmdClusterInfoDump(f, out))
```

Comment:
Suggest adding a hint of `dump` when printing results of `kubectl cluster-info`. Most parent commands will print help message with supported sub-commands, but `kubectl cluster-info` doesn't do that. WDYT?

Something like, 
```console
$ kubectl cluster-info
Kubernetes master is running at http://xxx
...
To further debug and diagnose cluster problems, use `kubectl cluster-info dump`.
```